p5Update.js

```diff
@@ -5,60 +5,78 @@
 const replace = require("replace-in-file");
 
 //Specify the release you want to update to
-const release = "0.9.0";
+const release = "0.10.2";
 
 const paths = ["./**/index.html"];
 const ignored_files = ["./_site/**/*.html"];
 
 const p5_js = {
   name: "p5.js",
   files: paths,
-  from: new RegExp("https://cdnjs.cloudflare.com/ajax/libs/p5.js/[0-9].[0-9].[0-9]/p5.js"),
-  to: `https://cdnjs.cloudflare.com/ajax/libs/p5.js/${release}/p5.min.js`,
+  from: new RegExp(
+    `https:\/\/cdn.jsdelivr.net\/npm\/p5@[0-99].[0-99].[0-99]\/lib\/p5.js`
+  ),
+  to: `https://cdn.jsdelivr.net/npm/p5@${release}/lib/p5.min.js`,
   ignore: ignored_files
 };
 const p5_sound_js = {
   name: "p5.sound.js",
   files: paths,
-  from: new RegExp("https://cdnjs.cloudflare.com/ajax/libs/p5.js/[0-9].[0-9].[0-9]/addons/p5.sound.js"),
-  to: `https://cdnjs.cloudflare.com/ajax/libs/p5.js/${release}/addons/p5.sound.min.js`,
+  from: new RegExp(
+    `https:\/\/cdn.jsdelivr.net\/npm\/p5@[0-99].[0-99].[0-99]\/lib\/addons\/p5.sound.js`
+  ),
+  to: `https://cdn.jsdelivr.net/npm/p5@${release}/lib/addons/p5.sound.min.js`,
   ignore: ignored_files
 };
 const p5_dom_js = {
   name: "p5.dom.js",
   files: paths,
-  from: new RegExp("https://cdnjs.cloudflare.com/ajax/libs/p5.js/[0-9].[0-9].[0-9]/addons/p5.dom.js"),
-  to: `https://cdnjs.cloudflare.com/ajax/libs/p5.js/${release}/addons/p5.dom.min.js`,
+  from: new RegExp(
+    `https:\/\/cdn.jsdelivr.net\/npm\/p5@[0-99].[0-99].[0-99]\/lib\/addons\/p5.dom.js`
+  ),
+  to: ``,
```

Comment:
Looks like you forgot to add the new path to p5.dom when you ran the script but otherwise looks good to me.

Reply:
p5.dom.js was merged into p5.js. See https://github.com/processing/p5.js/releases/tag/0.10.0

Reply:
Didn't know that thanks for pointing it out.